### DISCUSSION TITLE
derived files

### DISCUSSION THREAD

**Discussion by jh-source** [note286/xduts#3] [Ideas]
在[xduugtp](https://github.com/note286/xduugtp) 中，可以获得如下文件：
- `latexmkrc`：latexmk编译配置文件
- `xduugtp.bib`：参考文献样例文件
- `xduugtp.cls`：文档类文件
- `xduugtp.pdf`：文档类手册，建议重命名为`xduugtp-doc.pdf`
- `xduugtp.tex`：LaTeX源文件

在 [xduts](https://github.com/note286/xduts) 中，可以获得如下文件：
- `xduugthesis.cls`, Xidian University Undergraduate Thesis document class
- `xdufont.sty`, Xidian University Font package

[xduugtp](https://github.com/note286/xduugtp)提供了.tex和.bib的样例文件，可以直接在样例文件上编辑，IEEE提供的模板通常也会提供这些样例文件，提供样例文件对用户来说应该会更加易用。
![image](https://user-images.githubusercontent.com/86830694/163970267-ea7aba79-6ca0-4698-b315-be9c6c2592ae.png)

**Comment by note286** [comment]
xduts目前设计得非常简洁，最小工作示例在`xduts.pdf`中已经提供。
